.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+name: Node.js Package
+
+on:
+  release:
+    types: [created]
```

Comment:
what workflow you assume here?

Reply:
publish on release creation, like in old edumate_gateway